packages/twenty-front/src/pages/object-record/RecordIndexPage.tsx

```diff
@@ -28,44 +24,29 @@ export const RecordIndexPage = () => {
     objectNamePlural,
   });
 
-  const { findObjectMetadataItemByNamePlural } =
-    useObjectMetadataItemForSettings();
-
-  const { getIcon } = useIcons();
-  const Icon = getIcon(
-    findObjectMetadataItemByNamePlural(objectNamePlural)?.icon,
-  );
-
   const { createOneRecord: createOneObject } = useCreateOneRecord({
     objectNameSingular,
   });
 
   const recordIndexId = objectNamePlural ?? '';
-
   const setHotkeyScope = useSetHotkeyScope();
 
   const { setSelectedTableCellEditMode } = useSelectedTableCellEditMode({
     scopeId: recordIndexId,
   });
 
   const handleAddButtonClick = async () => {
-    await createOneObject?.({});
+    await createOneObject?.({
+      position: 0,
```

Comment:
I'm assuming there is already one element at position 0 when you load the page right? Will it still work? @charlesBochet

Reply:
It works but it's a hack. We will need to do a better job but this is fixing the current erratic component. I'll create a follow up ticket to properly implement the position that should be first / 2